src/cli/cli.cpp

```diff
@@ -1281,6 +1284,35 @@ void Interpreter::ProcessLinkQuality(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_FTD
+void Interpreter::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        const uint8_t *pskc = otThreadGetPSKc(mInstance);
```

Comment:
Do you need to read the PSKc for testing?
According to section 8.10.1.5, the PSKc has "Permissions: Write-only (such as a password entry field)".
The intention is that you should never extract this from a node but rather force the user to generate it from their secret passphrase and the XPANID salt.